docs/UserGuide.md

```diff
@@ -806,17 +660,51 @@ Unlike other entities, you may not delete a note by name. This is to minimise er
 Examples:
 * `listNotes` followed by `deleteNote 2` deletes the 2nd note in the SectresBook.
 
-[Back to Top](#sectresbook)
+[Back to Table of Contents](#table-of-contents)
+
+#### Locating a note by title : `findNote`
+
+Finds the notes whose titles match any of the given keywords. A filtered icon will show to the side of the Notes Panel to indicate that you are now viewing a filtered list.
+
+<img src="images/NotesFilteredIcon.png" width="256">
+
+Format: `findNote KEYWORD [MORE_KEYWORDS]`
+
+* The search is not case-sensitive. e.g. `meeting` will match `Meeting`
+* The order of the keywords does not matter. e.g. `Meeting Club` will match `Club Meeting`
+* Only the title is searched.
+* Only the full words will be matched. e.g. `Meetings` will not match `Meeting`
+* Keyword cannot contain special characters. However, numbers will be allowed.
+  * e.g. The keywords `?!` and `t-shirt` will not be allowed.
+  * e.g. `2` will match `shirt 2` but will not match `shirt2`
+* Keywords will ignore special characters.
+  * e.g. `Meetings` will match `Meetings!!!` and `Meetings 1`
+  * e.g. `shirt` will match `t-shirt`
+  * However, `tshirt` will not match `t–shirt` as special characters are ignored (`t-shirt` is now treated as two words, `t` and `shirt` with the special character `-` being treated as a spacing)
```

Comment:
This might be a bit too much points, some could be converted to prose form or put into the caution or note boxes

Reply:
You have updated